music-rpc.js

```diff
@@ -44,6 +45,24 @@ function getProps() {
   });
 }
 
+async function searchSong(props) {
+  const { artist, album } = props;
+  const options = {
+    term: encodeURI(decodeURI(`${artist} ${album}`)),
```

Comment:
> encodeURI(decodeURI(...))

For which reasons do we need to do this ?

Reply:
When there are Unicode characters like ł in polish when we will not do it, it will just throw error about characters